src/sknnr/transformers/_cca_transformer.py

```diff
@@ -3,13 +3,14 @@
 from sklearn.base import BaseEstimator, TransformerMixin
 from sklearn.utils.validation import FLOAT_DTYPES, check_is_fitted
 
+from . import ComponentReducerMixin
 from ._cca import CCA
 
 
-class CCATransformer(TransformerMixin, BaseEstimator):
+class CCATransformer(ComponentReducerMixin, TransformerMixin, BaseEstimator):
     @property
     def _n_features_out(self):
-        return self.cca_.eigenvalues.shape[0]
+        return self.n_components_
```

Comment:
This is obviously silly to just return another attribute with this property, but I think things would break with `_n_features_out` not defined, correct?

Reply:
Sweet, that works.  I had in the back of my brain that `_n_features_out` was critical, but it looks to only show up with the [`ClassNamePrefixFeaturesOutputMixin`](https://scikit-learn.org/stable/modules/generated/sklearn.base.ClassNamePrefixFeaturesOutMixin.html) that we had played around with, but ultimately wrote our own.  Thanks for the good thought!